### PR TITLE
Fix/mongodb rename

### PR DESCRIPTION
**Pull request by bigbitbus**
# Description
Change all mongo to mongodb atlas, as per request from MongoDB folks.

# Safety checklist
* [X ] This change is backwards compatible and safe to apply by existing users
* [X ] This change will NOT lead to data loss
* [ X] This change will NOT lead to downtime who already has an env/service setup

## How has this change been tested, beside unit tests?
Manual testing

### CODE REVIEW

**Review by NitinAgg — APPROVED** [review]

File: config/registry/common/modules/mongodb-atlas.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 ---
-title: "Atlas Mongo"
-linkTitle: "Atlas Mongo"
+title: "Mongdb Atlas"
```
Comment (legalosLOTR) [inline config/registry/common/modules/mongodb-atlas.md:2]: `title` and `linkTitle` need to be the same as the module name: `mongodb-atlas`

File: config/registry/common/modules/mongodb-atlas.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
+linkTitle: "Mongdb Atlas"
 date: 2021-10-12
 draft: false
 weight: 1
-description: Creates an Atlas Mongo database instance
+description: Creates an Mongdb Atlas database instance
```
Comment (legalosLOTR) [inline config/registry/common/modules/mongodb-atlas.md:7]: Typo in `Mongdb`

**Review by legalosLOTR — COMMENTED** [review]
Please make sure to test that the mongodb example that we demo'ed works after this change.

File: config/registry/common/modules/mongodb-atlas.md (hunk 1 above)
Comment (bigbitbus) [reply config/registry/common/modules/mongodb-atlas.md:2]: oh ok, I didnt realize that. will fix

**Pull request merged**